docs/operator/cn/02-10-update.md

```diff
@@ -0,0 +1,15 @@
+# Updating {Component Name}
```

Comment:
This should have a section for significant changes since the last release, depending on the release type. For patch updates, nothing is expected. For minor or major releases breaking changes are expected to be explained here.

This includes, but not limited to:
 - Mechanics changes (something worked X way, now works Y way)
 - New functionalities
 - Obsolete functionalities
 - Configuration changes
 - Updating pitfalls

Reply:
Aneta confirmed that "it's actually both the stable steps but also the specific for the concrete version update. As an example, you can have a look at the BAS update documentation:
https://help.sap.com/docs/OPSAWS/a57e20d1d0124a5db7cf6b68a4b60c06/51a431e8a20841e5a3c258bda526ac10.html?state=DRAFT". Also, in addition, we probably will also publish What's New.

Reply:
I proposed a section called "What's Changed". PTAL.